7_react/react1/week1/reactHomework/vite-project/src/App.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import TodoList from './components/TodoList/TodoList.jsx'
+import './App.css'
+
+function App() {
+  const todos = [
```

Comment:
Hey @MaryamAarhus, good job! Just one thing: I know you probably haven't covered lifecycle methods in React yet, but I just wanted to mention it as a fact for now. In general, keeping static variables, like the `todo` array, is considered bad practice. I know you haven't learned this yet; I just want you to keep it in mind for now. 🙂 Nevertheless, the solution is super simple just move the variable outside the Function as here
```
const todos = [ ...data ]
function App() {
```
Thats it! 

We can discuss this further if you are interested :)